### PR TITLE
Fix e2e tests - Closes #3261

### LINKED ISSUE #3261: Fix broken e2e tests after updating to SDK v4
### Expected behavior
All e2e tests should pass.

### Actual behavior
Some e2e tests fail because of the changes introduced as a result of the update to SDK v4.

### PR DESCRIPTION
**Pull request by UsamaHameed**
### What was the problem?
This PR resolves #3261

### CODE REVIEW

File: src/actions/transactions.js [hunk 1]
```diff
@@ -174,8 +175,14 @@ export const transactionBroadcasted = (transaction, callback = () => {}) =>
       data: transaction,
     });
 
+    console.log(transaction);
```
Comment (isalga) [inline src/actions/transactions.js:178]: Please remove this log

File: test/cypress/features/login.feature [hunk 2]
```diff
@@ -18,55 +18,14 @@ Feature: Login
...
     Given I am on Login page
-    When I click on settingsMenu
-    And I click on switchNetworksTrigger
-    And I click on closeDialog
     When I choose devnet
     When I enter first passphrase of genesis
     When I login
     Then I should be connected to devnet
-
-  Scenario: Log in to Mainnet (Network switcher is not enabled)
```
Comment (isalga) [inline test/cypress/features/login.feature:42]: Why is this removed?

File: test/cypress/features/wallet.feature [hunk 3]
```diff
@@ -6,11 +6,6 @@ Feature: Wallet
     Then I should see 30 transactions
     When I click show more
     Then I should see more than 30 transactions
-    Then I click filter incoming
-    Then I should see incoming transaction in table
```
Comment (isalga) [inline test/cypress/features/wallet.feature:10]: That should be working right?

File: test/cypress/features/login.feature (hunk 2 above)
Comment (UsamaHameed) [reply test/cypress/features/login.feature:42]: There were two tests testing the same thing.

File: test/cypress/features/wallet.feature (hunk 3 above)
Comment (UsamaHameed) [reply test/cypress/features/wallet.feature:10]: As far as I remember, we have removed those filter buttons.

Comment (reyraa) [reply test/cypress/features/wallet.feature:10]: yes, these tabs are removed.

**Pull request merged**